### ISSUE TITLE
Sentence analysis from example

### ISSUE THREAD

**Issue by rbleuse** [mvysny/aedict#920]
Hello Martin,

there is an entry error in the sentence analysis from the example https://tatoeba.org/eng/sentences/show/229756 :

ような should display the 様な entry (which is a kana-only), instead it displays the kanji 様.

<img src="https://user-images.githubusercontent.com/626863/83762354-bf5bb980-a677-11ea-9334-9d81538bc7fc.png" width="300" />

Android 10
Aedict 3.50.17
French dictionaries from June 1st

**Comment by mvysny** [comment]
Hi! The definition of the sentence `アレックスは、「クルミ」や、「紙」や、「とうもろこし」のような物の名前を言うことができる。` from `jpn_indices.csv` is as follows:

```
は|1 胡桃{クルミ}~ や 紙(かみ)  や 玉蜀黍{とうもろこし} の[01]{の} 様(よう){ような} 物(もの)[01] の 名前[01] を 言う 事(こと){こと} が[01] 出来る{できる}
```

The `様(よう){ような}` means that the word `様` appears in the sentence as `ような` yet the reading of the base form is `よう`. 

The analysis is currently limited in a way that it always shows the base form, which is 様. The reason is that the analysis simply shows a list of dictionary entries as they are present in the dictionary; the dictionary always lists words in their base form.

I'm not sure whether I can modify the entry, but perhaps I could add the information how the word appears in the sentence. So, for example, 様 would then show additional information "in sentence: ような".

**Comment by rbleuse** [comment]
I understand the limitation,
the problem is that it's not the correct word that is displayed with that kanji, that 様 doesn't make any sense.

In this screenshot, the correct kanji to display is the second one, but I suppose we can't do anything about that because we can't guess which one is the correct one :
<img src="https://user-images.githubusercontent.com/626863/89264925-07756900-d634-11ea-9f21-2225d68e7bf1.png" width="300" />

For 様な it would have been perfect ^^
<img src="https://user-images.githubusercontent.com/626863/89264969-178d4880-d634-11ea-91d0-6760b45b5afc.png" width="300" />

**Comment by mvysny** [comment]
> the problem is that it's not the correct word that is displayed with that kanji, that 様 doesn't make any sense.

Ah, right: the second entry "apparaisant" would make more sense. Yeah - Aedict can't really decide which one to use, and will use the more common one (which in this case is the incorrect one).

However, let me take a look whether I can modify the analyzer engine to try to search for 様+ような first, before falling back to a search for 様+よう.

**Comment by mvysny** [comment]
Actually I believe that the data set is incorrect in this case: it should have referenced the 様な word as follows: `様な(ような)` instead of `様(よう){ような}`. What do you think? If you agree that there may be a problem in the data set, could you please contact the Tatoeba guys to discuss a fix for this particular sentence please?

**Comment by rbleuse** [comment]
There are 635 sentences in that file that use 様(よう){ような}. Do you mean that all of them are wrong ?

I think the problem is that `Mr.; Mrs.; Miss; Ms` result is displayed whereas its reading is さま but not よう (I'm referring to the word, not the kanji)

**Comment by mvysny** [comment]
You're right: I've checked a couple of example sentences and they all use the form `様(よう){ような}`. So I definitely need to do something about it. Let me try :+1:

**Comment by mvysny** [comment]
Fixed in Aedict 3.50.18
